src/sdk/namespace_fs.js

```diff
@@ -3133,6 +3157,22 @@ class NamespaceFS {
         }
     }
 
+    /**
+     * _check_version_moved recieves key and version_id and checks if the version still exists in one of the optional locations
```

Comment:
Worth mentioning that it is out best effort?
(might be a case where it moved after this function returned and before the caller returned the value).